README.md

```diff
@@ -290,6 +290,8 @@ Any *backwards incompatible* change to either the Node.js Core Library API or Ap
 
 Any Modification to the ABI, dependencies or Native abstractions that require a developer to *modify code* and then recompile is considered a backwards incompatible change that must result in a *semver-major* version increase.
 
+Reversions of any changes landed into *mater* are to be handled just like any other type of pre- or post-release modification. Reversions that occur while generating a Release may happen only when a particular commit is interfering with the successful completion of the build, test and release process and the Release is intended to address a security vulnerability.
```

Comment:
> *mater*

*master*

--- 

I'm still quite hesitant on the reversion-while-release thing, I suggest we only put it in if consensus in #48 is for it.